packages/preset-create-react-app/package.json

```diff
@@ -24,15 +24,18 @@
   "dependencies": {
     "@storybook/node-logger": "^5.2.0",
     "@types/webpack": "^4.41.2",
+    "pnp-webpack-plugin": "^1.6.4",
     "react-docgen-typescript-loader": "^3.6.0",
-    "semver": "^7.1.1"
+    "semver": "^7.1.3"
   },
   "devDependencies": {
-    "@types/semver": "^6.2.0",
+    "@types/semver": "^7.1.0",
     "typescript": "^3.7.5"
   },
   "peerDependencies": {
     "@storybook/react": ">=5.2",
-    "react-scripts": ">=3.0.0"
+    "node-sass": "*",
```

Comment:
This is actually a dependency of `react-scripts` if you want to use Sass, and is not related to this preset in my opinion :)

Reply:
True, I added at the beginning of my work on Yarn 2 compatibility but cannot find any reason to keep it. I added a fixup commit and will rebase the branch properly after the PR gets validated ;)